README.md

```diff
@@ -1,3 +1,45 @@
-# setUp-webpack-boilerplate
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service
+
+![screenshot](./screenshot.PNG)
```

Comment:
- [ ] The link to the screenshot is broken, please kindly fix, as this will make your readme file look more professional.

Reply:
Hello, dear reviewer!
As you told me in the previous review,  I fixed that error! Kindly try again the `npm install` then `npm start` to run the application or just follow the link I add in my readme file.